site2/docs/cookbooks-retention-expiry.md

```diff
@@ -122,6 +128,8 @@ admin.namespaces().getRetention(namespace);
 
 You can control the allowable size of backlogs, at the namespace level, using *backlog quotas*. Setting a backlog quota involves setting:
 
+TODO: Expand on is this per backlog or per topic?
```

Comment:
And if this is per topic, how is it computed?  (See Slack conversation).

Reply:
this is per topic.

Reply:
Er, so to clarify:

Suppose I have 10 subscriptions on a topic.  They have not acknowledged the last 1000 messages, which are 1 kilobyte each.

Is the value that is being compared to the quota 1 megabyte or 10 megabytes?

Reply:
the backlog is the number of messages. so the value is 1000.